app/src/main/java/com/kryptgames/health/fitwithfriends/CreateProfile.java

```diff
@@ -0,0 +1,326 @@
+package com.kryptgames.health.fitwithfriends;
+
+import android.app.DatePickerDialog;
+
+import android.content.Intent;
+import android.graphics.Bitmap;
+import android.graphics.Color;
+import android.graphics.drawable.ColorDrawable;
+import android.net.Uri;
+import android.provider.MediaStore;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+import android.os.Bundle;
+import android.text.InputFilter;
+import android.text.Spanned;
+import android.text.TextUtils;
+import android.util.Log;
+
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.Button;
+import android.widget.DatePicker;
+import android.widget.EditText;
+
+import android.widget.Spinner;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.google.firebase.database.DataSnapshot;
+import com.google.firebase.database.DatabaseError;
+import com.google.firebase.database.DatabaseReference;
+import com.google.firebase.database.FirebaseDatabase;
+import com.google.firebase.database.Query;
+import com.google.firebase.database.ValueEventListener;
+import com.google.firebase.database.snapshot.Index;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.List;
+
+import de.hdodenhof.circleimageview.CircleImageView;
+
+public class CreateProfile extends AppCompatActivity {
+
+    public static final String TAG = "MainActivity";
```

Comment:
Your Class Name is CreateProfile and TAG is MainActivity make sure you name it consistently and always remember to make sure all Activity classes end with the name Activity